index.html

```diff
@@ -15,8 +15,15 @@
 <body>
     <form>
         1st Number : <input type="text" id="firstNumber" /><br>
-        2nd Number: <input type="text" id="secondNumber" /><br>
+        2nd Number: <input type="text" id="secondNumber" /><br><br>
         <input type="button" onClick="addition()" Value="Addition" />
+        <input type="button" onClick="subtraction()" value="Subtraction"/> 
```

Comment:
You did good all of this works but remember one of the concepts D.R.Y Don't repeat yourself.  This should have been a drop down box to select your operator rather then 4 independent buttons. Rather then 4 different functions you could have created just one function and used a switch statement which we learned in class last week.